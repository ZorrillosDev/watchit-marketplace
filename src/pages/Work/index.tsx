// react imports
import React from 'react';

// ===========================|| WORK ||=========================== //

const Work = (): JSX.Element => {
  return <span>How it work</span>;
};

export default Work;
