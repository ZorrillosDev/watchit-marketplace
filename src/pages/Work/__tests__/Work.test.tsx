import React from 'react'
import { shallow } from 'enzyme'
import Work from '../index'

/* eslint-disable no-undef */
describe('<Work />', () => {
  it('should render', () => {
    const component = shallow(<Work />)
    expect(component).toMatchSnapshot()
  })
})
