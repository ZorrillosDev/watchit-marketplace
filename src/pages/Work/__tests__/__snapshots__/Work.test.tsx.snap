// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Work /> should render 1`] = `ShallowWrapper {}`;
